doc/news/changes/minor/20180602BrunoTurcksin

```diff
@@ -0,0 +1,4 @@
+New: Add function print_numpy_array to SparseMatrix and LinearAlgebra::Vector to
```

Comment:
If you write it as follows, then doxygen will automatically link to the new function:
```suggestion
New: Add function to SparseMatrix::print_numpy_array() and LinearAlgebra::Vector::print_numpy_array() to
```
But see the note about function names below.